client/src/App.js

```diff
@@ -20,6 +24,26 @@ export const PATHS = {
   register: "/register",
 }
 
+/*
+  STORE SETUP
+*/
+
+const mapStateToProps = state => {
+  return ({
+    schoolName: state.schoolName,
+    count: state.countState.count
+  })
+}
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
A `dispatch` is used to trigger an action. The payload passed in is used to make changes to the store state in a reducer.